src/app/account-settings/components/settings-security-alternate-sign-in/settings-security-alternate-sign-in.component.html

```diff
@@ -0,0 +1,63 @@
+<app-settings-panels-data *ngIf="accounts$ | async as accounts">
+  <div i18n="@@account.youCanSingIntoOrcid">
+    You can sign into ORCID using the personal and institutional accounts you
+    have linked to your ORCID record.
+  </div>
+
+  <a
+    rel="noopener noreferrer"
+    href="https://www.figma.com/exit?url=https%3A%2F%2Fsupport.orcid.org%2Fhc%2Fen-us%2Farticles%2F360006894354-Sign-into-ORCID-using-your-social-media-account"
+    i18n="@@account.learnMoreAboutUsing"
+  >
+    Learn more about using alternate accounts to sign in to ORCID
+  </a>
+  <table
+    *ngIf="accounts.length"
+    mat-table
+    [dataSource]="accounts"
+    class="mat-elevation-z0"
+  >
+    <ng-container matColumnDef="provider">
+      <th mat-header-cell *matHeaderCellDef i18n="@@account.account">
+        Account
+      </th>
+      <td mat-cell *matCellDef="let element">
+        {{ element?.id?.providerid | titlecase }}
+      </td>
+    </ng-container>
+
+    <ng-container matColumnDef="email">
+      <th mat-header-cell *matHeaderCellDef i18n="@@account.associatedEmail">
+        Associated email
+      </th>
+      <td mat-cell *matCellDef="let element">{{ element.email }}</td>
+    </ng-container>
+
+    <ng-container matColumnDef="granted">
+      <th mat-header-cell *matHeaderCellDef i18n="@@account.accessGranted">
+        Access granted
+      </th>
+      <td mat-cell *matCellDef="let element">
+        {{ element.dateCreated | date }}
+      </td>
+    </ng-container>
+
+    <ng-container matColumnDef="actions">
+      <th mat-header-cell *matHeaderCellDef></th>
+      <td mat-cell *matCellDef="let element">
+        <button (click)="delete(element)" mat-icon-button>
+          <mat-icon>delete</mat-icon>
+        </button>
+      </td>
+    </ng-container>
+
+    <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
+    <tr mat-row *matRowDef="let row; columns: displayedColumns"></tr>
+  </table>
+  <div *ngIf="!accounts.length" class="emptyAccounts">
+    <mat-divider class="emptyAccounts"></mat-divider>
```

Comment:
is this mat-divider correct having the same class as the parent div

Reply:
yes that is what we need there... but I am changing the name so is not confusing